client/src/App.js

```diff
@@ -112,23 +139,30 @@ export default function App() {
           <QueueDisplay items={queue} leaveQueueFunction={displayExitModal} />
         </div>
       )}
-      <div className='queueForm'>
-        <input
-          type='text'
-          placeholder='Dine initialer'
-          className='textinput'
-          ref={inputRef}
-          onKeyUp={handleKeyPress}
-        ></input>
-        <br></br>
-        <button className='button' onClick={enterQueue}>
+      <form className='queueForm' onSubmit={handleSubmit}>
+        <div>
+          <input
+            type='text'
+            placeholder='Dine initialer'
+            className='textinput'
+            ref={initialsInputRef}
+          />
+          <input
+            type='number'
```

Comment:
consider `type='time` for better formatting. Also allows for non-whole number times to be added